java/src/plume/Intern.java

```diff
@@ -856,7 +857,7 @@ public SequenceAndIndices(T seq, int start, int end) {
     }
 
     @SuppressWarnings("unchecked")
-    /*@Pure*/ public boolean equals(/*@Nullable*/ Object other) {
+    /*@Pure*/ public boolean equals (/*>>>@GuardSatisfied SequenceAndIndices<T> this,*/ /*@GuardSatisfied*/ /*@Nullable*/ Object other) {
```

Comment:
Fixed. Sorry about that - I must have made a mistake in the find/replace expressions.